### PR TITLE
Github code coverage integration via codecov

### PR DESCRIPTION
**Pull request by hardbyte**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ install:
   # We need wheel installed to build wheels
   - "%PYTHON%\\python.exe -m pip install -U wheel setuptools"
   # We need our development dependencies for testing
-  - "%PYTHON%\\python.exe -m pip install -U -r requirements.txt"
+  - "%PYTHON%\\python.exe -m pip install -U -r requirements.txt codecov"
```
Comment (nbgl) [inline .appveyor.yml]: Minor: It’s not immediately clear why this is a valid command. I would suggest splitting it into two.

File: .travis.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ python:
   - '2.7'
   - '3.5'
   - '3.6'
+  - '3.7-dev'
```
Comment (nbgl) [inline .travis.yml]: Python 3.7 is out now.

File: .travis.yml [hunk 3]
```diff
@@ -30,9 +31,10 @@ matrix:
 
 install:
   - if [ "${MYPY_TYPING}" == "1" ]; then travis_retry pip install mypy; fi
-  - travis_retry pip install -r requirements.txt
+  - travis_retry pip install -r requirements.txt codecov
```
Comment (nbgl) [inline .travis.yml]: Minor: It’s not immediately clear why this is a valid command. I would suggest splitting it into two.

File: .travis.yml (hunk 2 above)
Comment (hardbyte) [reply .travis.yml]: Not supported in travis when I checked yesterday though

Comment (hardbyte) [reply .travis.yml]: But trying again [now](https://travis-ci.org/n1analytics/clkhash/builds/399429288?utm_source=github_status&utm_medium=notification)

Comment (hardbyte) [reply .travis.yml]: 👎 https://travis-ci.org/n1analytics/clkhash/jobs/399429292

**Pull request merged**